src/qt/clientmodel.cpp

```diff
@@ -284,7 +272,11 @@ void ClientModel::subscribeToCoreSignals()
         [this](bool network_active) {
             Q_EMIT networkActiveChanged(network_active);
         });
-    m_handler_notify_alert_changed = m_node.handleNotifyAlertChanged(std::bind(NotifyAlertChanged, this));
+    m_handler_notify_alert_changed = m_node.handleNotifyAlertChanged(
+        [this]() {
+            qDebug() << "NotifyAlertChanged";
+            Q_EMIT alertsChanged(getStatusBarWarnings());
```

Comment:
9196d8a92837a077f65b3660d5cff523728e7db6

Wrong commit message, should be AlertChanged.

Also using auto connection:
https://github.com/bitcoin-core/gui/blob/747cdf1d652d8587e9f2e3d4436c3ecdbf56d0a5/src/qt/overviewpage.cpp#L254
https://github.com/bitcoin-core/gui/blob/747cdf1d652d8587e9f2e3d4436c3ecdbf56d0a5/src/qt/rpcconsole.cpp#L663

Reply:
Thanks! [Updated](https://github.com/bitcoin-core/gui/pull/581#issuecomment-1100709598).